glific/glific-frontend
# API details from current window.location

## Summary 💬 

API details to be based on current frontend url

## Test Plan

Not needed



## How to test? - Please test thoroughly!!
1. Add any domain to your `/etc/hosts` file. For example `127.0.0.1   glific.test.com`
2. Verify that the backend app is accessible at http://glific.test.com:4000
3. Run the front-end server using the following command
```sh
> HOST=glific.test.com yarn start
```
4. Navigate to http://glific.test.com:3000
